core/class-maxi-style-cards.php

```
@@ -2308,6 +2561,132 @@ private static function get_maxi_sc_styles($args)
                 $added_response .= "{$target}.current-menu-item:hover { background-color: var(--maxi-{$style}-menu-item-sub-bg-hover); }";
             }
 
+            $container_default_size_settings = [
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$container_default_size_settings'. (undefined)

(UnusedLocalVariable)